### ISSUE TITLE
training procedure and pretrained  models in models

### ISSUE THREAD

**Issue by antonyvan** [DeformableFriends/NeuralTracking#13]
Hello,
Thank you for sharing your work to public. I am currently trying to run your code.  I got several questions regarding training.

1)  I see in options.py,  line 25-32, where we can define to use  there are several pretrained_model, 
model_module_to_load is for choosing  to load pretrained flownet or full model. and correspondingly you provided chairs_things, model_A, model_B, model_C for model_name. Are they all full model ? Are these models all full models trained by you?  what is the difference among then ? 

2) For the training, should I train the network by following your procedures from "0_flow" , then "s_solver" , then "2_mask" and then "3_refine".  If doing it this way, I should update the saved_model in line 31 to the newly trained model. right ? 

3) Could I just train the network in end--to-end way  by setting mode =="3_refine" ? 

4) I find that when I train the network with your provided pretrained model, the loss basically stays at 24-30 and does not reduce, even I set model_modele_to_load = "only_flow_net". Does that mean your pretrained models are  already perfect ? 
   
Thanks a million for your work and your time.
Best regards
